.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,7 +9,7 @@ assignees: ''
 
 ### Describe the bug
 
-A clear and concise description of what the bug is. Include CLI version by typing `gh actions-cache --version` and extension version by typing `gh actions-cache --version`. <!--TODO: Implement version printing for the cache -->
+A clear and concise description of what the bug is. Include CLI version by typing `gh --version.
```

Comment:
We need the version of gh cli as well as the extension here. 
Extension version can be found using this command (until we support it as a flag in the extension itself):
`gh extensions list | grep actions-cache`

Reply:
Done.